simple/stats/schema_constants.py

```diff
@@ -34,6 +34,16 @@
 PREDICATE_SEARCH_DESCRIPTION = "searchDescription"
 PREDICATE_MEASURED_PROPERTY = "measuredProperty"
 PREDICATE_DESCRIPTION = "description"
+PREDICATE_UNIT = "unit"
+PREDICATE_SCALING_FACTOR = "scalingFactor"
+PREDICATE_MEASUREMENT_METHOD = "measurementMethod"
+PREDICATE_OBSERVATION_PERIOD = "observationPeriod"
```

Comment:
(minor, not worth changing for now, and subjective) I feel the constant to too long for readability. Might just remove "PREDICATE_" everywhere